tpot/tpot.py

```diff
@@ -452,7 +443,8 @@ def _compile_to_sklearn(self, expr):
         return eval(sklearn_pipeline, self.operators_context)
 
     def _evaluate_individual(self, individual, features, classes):
-        """Determines the `individual`'s fitness according to its performance on the provided data
+        """Determines the `individual`'s fitness according to its performance on
+        the provided data
```

Comment:
Just a note: The initial short description in the docstrings should be on one line only.